README.md

```diff
@@ -18,97 +18,39 @@
-
-The only executables included in the playbook are open-sourced [here](https://github.com/Atlas-OS/utilities) under [GPLv3](https://github.com/Atlas-OS/utilities/blob/main/LICENSE), with the hashes being identical to the releases. Everything else is in plain text.
-
-### ✅ Improved Performance
-Some tweaked systems on the internet have tweaked Windows too much, breaking compatibility for main features such as Bluetooth, Wi-Fi, and so on.
-Atlas is on the sweet point. It aims at getting more performance while maintaining a good level of compatibility.
-
-Some of the many changes that we have made to improve Windows are listed below:
-- Customized power scheme
-- Reduced amount of services and drivers
-- Disabled audio exclusive
-- Disabled unneeded devices
-- Disabled power saving (for personal computers)
-- Disabled performance-hungry security mitigations
-- Automatically enabled MSI mode on all devices
-- Optimized boot configuration
-- Optimized process scheduling
-
-### 🔒 Legal
-Many custom Windows OSes distribute their systems by providing a tweaked ISO of Windows. Not only does it violate [Microsoft's Terms of Service](https://www.microsoft.com/en-us/Useterms/Retail/Windows/10/UseTerms_Retail_Windows_10_English.htm), but it is also not a safe way to install.
-
-Atlas partnered with the Ameliorated team to provide users with a safer and legal way to install, using [AME Wizard](https://ameliorated.io). With it, Atlas fully complies with [Microsoft's Terms of Service](https://www.microsoft.com/en-us/Useterms/Retail/Windows/10/UseTerms_Retail_Windows_10_English.htm).
-
+- [Branding](https://docs.atlasos.net/branding/)
+- [FAQ & Common Issues](https://docs.atlasos.net/faq-and-troubleshooting/removed-features/)
+
+Want to learn more about Atlas? Visit our [🌐 Website](https://atlasos.net/) and [📚 Documentation](https://docs.atlasos.net/).
+
+## 🤔 What is Atlas?
+Atlas is an open-source project that enhances Windows by eliminating factors that negatively impact gaming performance. It's the perfect choice for reducing system and network latency, and input lag, while maintaining system privacy and prioritizing performance.
+  
+## 👀 Why Atlas?
+### 🔒 Enhanced Privacy
+Atlas removes all types of tracking embedded within Windows and implements numerous group policies to minimize data collection. However, it cannot ensure the security for things outside the scope of Windows, such as browsers and other third-party applications.
+  
+### 🛡️ Increased Security
+Unlike other custom Windows ISOs, Atlas is safer as it doesn't require downloading a modified Windows ISO from the internet. Atlas also includes optional security features (e.g. Defender, Update) that other custom Windows OSs often strip.
```

Comment:
> Unlike other custom Windows ISOs

This should be reworded to 'Unlike custom Windows ISOs' as Atlas isn't a custom ISO. This should also be expanded upon by stating that Atlas & AME Wizard is a lot easier to audit compared to an ISO, with AME Wizard's backend being open source and Atlas mostly being a plain text 7z/zip archive.

> Atlas also includes optional security features (e.g. Defender, Update) that other custom Windows OSs often strip.

This is very good to mention, but Defender should be changed to Windows Defender and 'Update' should be changed to Windows Update. You should also note that Atlas has CPU mitigations and UAC enabled by default.